code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -9,8 +9,6 @@
 	health = 25
 	maxHealth = 25
 
-	radio_key = /obj/item/device/encryptionkey/headset_eng
-	radio_channel = "Engineering"
```

Comment:
Why are you removing this? Even if you remove the nag() function, player floorbots still have a use for talking on the Engineering channel.